### ISSUE TITLE
Texture Scale goes haywire when parts are moved

### ISSUE THREAD

**Issue by Blue20Boy17** [JannisX11/blockbench#134]
After I get the texture mapping perfect, I can save my model, reopen it, and try to move or resize parts, and the texture mapping goes haywire. Here's an example of before and after it being moved...

![after](https://user-images.githubusercontent.com/40492047/44962252-7e0b0880-aeeb-11e8-8a25-9e116a075fb3.png)
![before](https://user-images.githubusercontent.com/40492047/44962253-7e0b0880-aeeb-11e8-8dfd-75bf56c17c71.png)

**Comment by Nianor** [comment]
I can also confirm this is an issue. It seems to be something with his models. I've never had it with any of my models, but as soon as I opened one of his, it started cropping up.

**Comment by JannisX11** [comment]
Note to self: Make auto-UV less confusing.

**Comment by Nianor** [comment]
I may be wrong, but I don't think this is an auto-UV problem. Or, if it is, it's an actual problem, and not auto-UV working as intended. I checked, and the way it remaps things when I use auto-UV is not the same way it remapped them in this issue. I manually used auto-UV on one of the affected parts, and the resulting map was different.

**Comment by JannisX11** [comment]
If it is an issue with the model files, could you share one for me to test?

**Comment by Blue20Boy17** [comment]
yeah, here's one of my models and texture
![whitedogbeast](https://user-images.githubusercontent.com/40492047/45264082-b4251c80-b404-11e8-82de-2337a1b6066c.png)

**Comment by JannisX11** [comment]
Yeah, that's an auto UV issue. For now, you can disable "Export Groups" in the settings to export the model.

**Comment by Blue20Boy17** [comment]
hear that nianor?

**Comment by Blue20Boy17** [comment]
Me and Nianor both tried this, and it didn't work.
